python/ray/serve/api.py

```diff
@@ -1738,6 +1738,7 @@ def run(
     host: str = DEFAULT_HTTP_HOST,
     port: int = DEFAULT_HTTP_PORT,
     driver: Optional[Deployment] = None,
+    default_route_prefix: Optional[str] = "/",
```

Comment:
this parameter is unused, forgot to pass in?

Reply:
and test this path?

Reply:
driver.options from user looks more reasonable to me, updated